erpnext/stock/doctype/quality_inspection_reading/quality_inspection_reading.json

```diff
@@ -215,7 +215,7 @@
  "idx": 1,
  "istable": 1,
  "links": [],
- "modified": "2021-01-07 21:56:40.235579",
+ "modified": "2021-01-07 18:16:53.978410",
```

Comment:
timestamp must be later than previous timestamp. Else your JSON changes wont be applied. Either shift the date or the hour ( > 21 hours)